common/src/main/java/com/wynntils/core/webapi/profiles/item/ItemTier.java

```diff
@@ -87,13 +87,22 @@ public ChatFormatting getChatFormatting() {
         return chatFormatting;
     }
 
+    public boolean isHighlightEnabled() {
+        try {
+            return highlightEnabled.call();
+        } catch (Exception e) {
+            e.printStackTrace();
```

Comment:
1) Use logger instead.
2) make return in the catch block, then the comment is unnecessary.